components/CreateCollectiveMiniForm.js

```diff
@@ -319,6 +327,32 @@ const CreateCollectiveMiniForm = ({
                   />
                 )}
               </StyledInputField>
+              {isOrganization && (
+                <StyledInputField
+                  name="legalName"
+                  htmlFor="legalName"
+                  label={formatMessage(msg.legalName)}
+                  mt={3}
+                  value={values.legalName}
+                  isPrivate
+                  hint={
+                    <FormattedMessage
+                      id="legalName.description"
+                      defaultMessage="The legal name is private and shared with the hosts for donation receipts, tax forms and when you submit and expense. This name is not displayed publicly and it must be your legal name."
```

Comment:
```suggestion
                      defaultMessage="The legal name is private and shared with the hosts for donation receipts, tax forms and when you submit an expense. This name is not displayed publicly and it must be your legal name."
```

Reply:
Maybe also use the word `host` instead of `hosts` since the organization has one host not many? 🤔

Reply:
> Maybe also use the word `host` instead of `hosts` since the organization has one host not many? thinking

The organization itself has no host. It financially contributes to many collectives that each have one fiscal host, therefore the legal name of the organization can be shared with multiple hosts.

Example: Facebook contributes to Babel then NumPy, its legal name is shared with 2 fiscal hosts: Open Source Collective and NumFocus.

Reply:
Ah, that makes sense. Thanks for the explanation. 👍🏽